### PR TITLE
removing support of old classifiers in download

### PR DESCRIPTION
**Pull request by ronykoz**
## Status
- [x] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

removing support of old classifiers in download

### CODE REVIEW

File: demisto_sdk/commands/download/downloader.py [hunk 1]
```diff
@@ -188,7 +188,8 @@ def handle_list_files_flag(self) -> bool:
         """
         if self.list_files:
             self.all_custom_content_objects = self.get_custom_content_objects()
-            list_files = [[cco['name'], cco['entity'][:-1]] for cco in self.all_custom_content_objects]
+            list_files = [[cco['name'], cco['entity'][:-1]] for cco in self.all_custom_content_objects
+                          if cco.get('name')]
```
Comment (guyfreund) [inline demisto_sdk/commands/download/downloader.py:192]:
```suggestion
            list_files = [[cco['name'], cco['entity'][:-1]] for cco in self.all_custom_content_objects
                          if 'name' in cco]
```

File: demisto_sdk/commands/download/downloader.py [hunk 2]
```diff
@@ -321,7 +322,7 @@ def build_custom_content(self) -> None:
         for input_file_name in self.input_files:
             input_file_exist_in_cc: bool = False
             for custom_content_object in custom_content_objects:
-                name = custom_content_object['name']
+                name = custom_content_object.get('name', 'N/A')
```
Comment (guyfreund) [inline demisto_sdk/commands/download/downloader.py:325]: what is `N/A`?

File: demisto_sdk/commands/download/downloader.py (hunk 1 above)
Comment (ronykoz) [reply demisto_sdk/commands/download/downloader.py:192]: name might be empty

File: demisto_sdk/commands/download/downloader.py (hunk 2 above)
Comment (ronykoz) [reply demisto_sdk/commands/download/downloader.py:325]: Not available

**Pull request merged**